pallets/automation-time/src/lib.rs

```diff
@@ -245,6 +285,49 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		/// Schedule a task to fire an event with a custom message.
+		///
+		/// Before the task can be scheduled the task must past validation checks.
+		/// * The transaction is signed
+		/// * The provided_id's length > 0
+		/// * The time is valid
+		/// * 
+		///
+		/// # Parameters
+		/// * `provided_id`: An id provided by the user. This id must be unique for the user.
+		/// * `time`: The unix standard time in seconds for when the task should run.
+		/// * `message`: The message you want the event to have.
+		///
+		/// # Errors
+		/// * `InvalidTime`: Time must end in a whole minute.
+		/// * `PastTime`: Time must be in the future.
+		/// * `DuplicateTask`: There can be no duplicate tasks.
+		/// * `TimeSlotFull`: Time slot is full. No more tasks can be scheduled for this time.
+		/// * `InvalidAmount`: Amount has to be larger than 0.1 OAK.
+		/// * `InvalidRecipient`: Recipient must be a valid address.
+		/// * `TransferToSelf`: Sender cannot transfer money to self.
+		#[pallet::weight(<T as Config>::WeightInfo::schedule_transfer_task_existing_slot())]
+		pub fn schedule_transfer_task(
+			origin: OriginFor<T>,
+			provided_id: Vec<u8>,
+			time: UnixTime,
+			recipient_id: T:: AccountId,
+			amount: T::Balance,
+		) -> DispatchResult {
+			let who = ensure_signed(origin)?;
+			let recipientExists = account_exists(recipient_id);
+			
+			// check for greater than existential deposit
+			ensure!(amount < T::ExistentialDeposit::get(), Err(<Error<T>>::InvalidAmount)?);
```

Comment:
weird behavior on ensure, see here: https://github.com/OAK-Foundation/OAK-blockchain/pull/74#discussion_r804364321

Reply:
also, i'm getting an error on the `<` comparator: 
```
binary operation `<` cannot be applied to type `<T as pallet::Config>::Balance`
the trait `std::cmp::PartialOrd` is not implemented for `<T as pallet::Config>::Balance`
```

Balance is set as u128. u128 should have comparators set. but it seems like it's not quite understanding that - maybe cause it's a generic Config and it's not sure what's getting inserted into the Balance slot. so in this case, is there any alternative to implementing the `std::cmp::PartialOrd` trait? cause i don't really wanna do that haha

Reply:
i guess i could just set T::ExistentialDeposit type to u128 rather than Self::Balance. but this seems lazy

Reply:
RESOLVED: idk why, it disappeared on its own 🤷